libraries/chain/asset_evaluator.cpp

```diff
@@ -483,7 +489,21 @@ void_result asset_update_evaluator::do_apply(const asset_update_operation& o)
          d.cancel_order(*itr);
    }
 
-   d.modify(*asset_to_update, [&](asset_object& a) {
+   // For market-issued assets, if core change rate changed, update flag in bitasset data
```

Comment:
Not sure whether this is applicable for us. Can you confirm ?

Reply:
One of the optimizations,  introduced in b67f0523, is the tracking of the current exchange rates, instead of iterating of all `asset_object`s for every new block to check if the feed is expired. This optimization works when we have many assets.